web/src/redux/persistent/cells/reducer.ts

```diff
@@ -21,12 +21,12 @@ export default function (state: State = defaultState, action: Action<CellIdStrin
     case SET_PROFILES_CELL_ID:
       return {
         ...state,
-        profiles: payload,
+        profiles: payload as CellIdString,
```

Comment:
The return type of this reducer was confused or saw things as ambiguous without this type annotation, due to the necessary use of the union type in the action argument